### PR TITLE
Add a way to override the identity

### PR DESCRIPTION
**Pull request by msm-code**

### CODE REVIEW

File: karton/config_extractor/config_extractor.py [hunk 1]
```diff
@@ -83,6 +83,11 @@ def args_parser(cls):
             default=[],
             nargs="+",
         )
+        parser.add_argument(
+            "--identity",
+            help="Override the default karton identity",
+            default=None,
```
Comment (chivay) [inline karton/config_extractor/config_extractor.py:89]: AFAIK setting the deafult to None is redundant :thinking:

**Review by chivay — COMMENTED** [review]
Typo: `overide` -> `override`

File: karton/config_extractor/config_extractor.py (hunk 1 above)
Comment (msm-code) [reply karton/config_extractor/config_extractor.py:89]: Weird, I've thought the default is to raise an exception. But I've tested it with some code (and psrok checked in the docs) and you're right

Comment (psrok1) [reply karton/config_extractor/config_extractor.py]: Ref: https://docs.python.org/3/library/argparse.html#default
```
The default keyword argument of add_argument(), whose value defaults to None [...]
```

File: karton/config_extractor/config_extractor.py [hunk 2]
```diff
@@ -83,6 +83,10 @@ def args_parser(cls):
             default=[],
             nargs="+",
         )
+        parser.add_argument(
+            "--identity",
+            help="Override the default karton identity",
```
Comment (psrok1) [inline karton/config_extractor/config_extractor.py:88]:
```suggestion
            help="Override the default Karton identity",
```
Capitilized version is used in docstring

**Review by psrok1 — APPROVED** [review]
Looks ok :+1:

**Pull request merged**